### DISCUSSION TITLE
dbm docker image

### DISCUSSION THREAD

**Discussion by Craillon** [EdurtIO/incubator-dbm#96] [Q&A] [answered]
Hi and I would like to know, is there a docker image of DBM?

**Comment by qianmoQ** [comment, marked as answer]
Since it is a GUI client tool, no friendly solution has been found to run it on Docker

**Reply by qianmoQ** [reply]
> Hi and I would like to know, is there a docker image of DBM?

> Since it is a GUI client tool, no friendly solution has been found to run it on Docker

If you have a good solution you can submit a PR for subsequent staff to use
